### PR TITLE
Fix docker build

### PR DESCRIPTION
**Pull request by pgebheim**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,12 +44,12 @@
     "release:major": "npm version major && git push && git push --tags && npm publish",
     "docker:build": "scripts/docker/build.sh",
     "docker:build-and-push": "npm run docker:build && scripts/docker/push.sh",
-    "docker:pull": "docker pull augurproject/dev-node-parity:latest && docker pull augurproject/dev-node-geth:latest && docker pull augurproject/dev-pop-geth:core-$(npm view augur-core version) && docker pull augurproject/dev-pop-normtime-geth:latest",
+    "docker:pull": "docker pull augurproject/dev-node-parity:latest && docker pull augurproject/dev-node-geth:latest && docker pull augurproject/dev-pop-geth:core-$(node scripts/package-version.js augur-core) && docker pull augurproject/dev-pop-normtime-geth:latest",
     "docker:geth": "docker run -e GETH_VERBOSITY=4 -it -p 8545:8545 -p 8546:8546 -v $HOME/docker:/geth/chain augurproject/dev-node-geth:latest",
```
Comment (bthaile) [inline package.json]: augurproject/dev-pop-normtime-geth:latest",
do we want to get rid of ;latest for normtime as well?

File: scripts/docker/build.sh [hunk 2]
```diff
@@ -1,17 +1,17 @@
 #!/bin/bash
 
-TAG=$(npm view augur-core version)
+TAG=core-$(node scripts/package-version.js augur-core)
 
 # Make sure you're up to date
-docker pull augurproject/dev-geth-node:latest
+docker pull augurproject/dev-node-geth:latest
```
Comment (bthaile) [inline scripts/docker/build.sh]: nice catch

**Comment by tinybike** [comment]
Lint is failing @bthaile

**Comment by bthaile** [comment]
Fixed linting errors.
Added a >npm run core:version to spit out augur-core version to help manually sync up augur-node and augur UI.

**Comment by bthaile** [comment]
got approval of changes for npm run core:version from @pgebheim. Merging

**Pull request merged**